### PR TITLE
Migrated our Solana Vault address to handle an upgraded protocol

### PR DESCRIPTION
**Pull request by Elaixzar**
Migrated our Solana Vault address to handle an upgraded protocol. 

Tested new code and looks to work.

Let me know if you need any additional information.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/glitter-finance exports TVL: 
        
 
 
    solana                    124.92 k
    algorand                  34.82 k
    
    total                    159.75 k

File: projects/glitter-finance/index.js [hunk 1]
```diff
@@ -21,7 +21,10 @@ async function solana_tvl() {
     const tvlResult = {};
 
     //Get sol balance from the SOL vault
-    const amount_sol = await getSolBalance(SOLANA_VAULT);
+    var amount_sol = await getSolBalance(SOLANA_VAULT);
+    if (!amount_sol || amount_sol == 'NaN') {
```
Comment (g1nt0ki) [inline projects/glitter-finance/index.js:25]: @Elaixzar thanks for the PR, it looks good except for this check here, if some error occurs we would rather it fail than return 0 which will result in wrong tvl, can you remove it

Comment (g1nt0ki) [reply projects/glitter-finance/index.js:25]: okay, I am merging it and fixing it.

Comment (Elaixzar) [reply projects/glitter-finance/index.js:25]: Looks like you already merged it, but noted for future reference.

**Pull request merged**